ts/Series/TiledWebMap/TiledWebMapSeries.ts

```diff
@@ -0,0 +1,698 @@
+            { provider } = options;
+
+        if (transformGroups) {
+            transformGroups.forEach((group): void => {
+                if (Object.keys(group).length !== 0) {
+                    group.destroy();
+                }
+            });
+            this.transformGroups = [];
+        }
+
+        if (
+            mapView &&
+            !defined(mapView.options.projection) &&
+            provider &&
+            provider.type
+        ) {
+            const ProviderDefinition = TilesProvidersRegistry[provider.type];
+
+            if (ProviderDefinition) {
+                const def = new ProviderDefinition(),
+                    { initialProjectionName: providerProjectionName } = def;
+
+                mapView.update({
+                    projection: {
+                        name: providerProjectionName
+                    }
+                });
+            }
+        }
+
+        super.update.apply(series, arguments);
+    }
+}
+
+addEvent(Chart, 'beforeMapViewInit', function (e: any): boolean {
+    const twm: TiledWebMapSeriesOptions =
+        (this.options.series || []).filter(
+            (s: any): boolean => s.type === 'tiledwebmap')[0],
+        { geoBounds } = e;
+
+    if (twm && twm.provider && twm.provider.type && !twm.provider.url) {
+        const ProviderDefinition =
+            TilesProvidersRegistry[twm.provider.type];
+
+        if (!defined(ProviderDefinition)) {
+            error(
+                'Provider cannot be reached.',
+                false
+            );
+        } else {
+            const def = new ProviderDefinition(),
+                { initialProjectionName: providerProjectionName } = def;
+
+            if (this.options.mapView) {
+                if (geoBounds) {
+                    const { x1, y1, x2, y2 } = geoBounds;
+                    this.options.mapView.recommendedMapView = {
+                        projection: {
+                            name: providerProjectionName,
+                            parallels: [y1, y2],
+                            rotation: [-(x1 + x2) / 2]
+                        }
+                    };
+                } else {
+                    this.options.mapView.recommendedMapView = {
+                        projection: {
+                            name: providerProjectionName
+                        }
+                    };
+                }
+            }
+
+            return false;
+        }
+    }
+
+    return true;
+});
+
+/* *
+ *
+ *  Prototype properties
+ *
+ * */
+
+interface TiledWebMapSeries {
+    // type: string;
+}
+
+/* *
+ *
+ *  Registry
+ *
+ * */
+declare module '../../Core/Series/SeriesType' {
+    interface SeriesTypeRegistry {
+        tiledwebmap: typeof TiledWebMapSeries;
+    }
+}
+
+SeriesRegistry.registerSeriesType('tiledwebmap', TiledWebMapSeries);
+
+/* *
+ *
+ *  Default export
+ *
+ * */
+
+export default TiledWebMapSeries;
+
+/* *
+ *
+ *  API options
+ *
+ * */
+
```

Comment:
I tried to build API docs locally but got some errors. It might be an issue on my side, but left you a comment for you to double-check.

Reply:
I double-checked that and everything is working correctly.